src/multi-level-memoizee.ts

```diff
@@ -2,9 +2,10 @@ import type { types } from '@balena/pinejs';
 import * as _ from 'lodash';
 import type { Options as MemoizeeOptions } from 'memoizee';
 import primitiveKey = require('memoizee/normalizers/primitive');
-import { SECONDS } from './config';
 import { createMultiLevelStore, Defined, MultiStoreOpt } from '.';
 
+export const SECONDS = 1000;
```

Comment:
Not exporting APIs *by default* -- that is, unless it's absolutely necessary -- is typically much better in the long term.

If you're set on exporting this specific constant though, I'd be interested in understanding why.

Reply:
That was a copy-paste error, good catch!